README.md

```diff
@@ -49,7 +49,7 @@ For examples of the kinds of bugs we're looking for, see [SECURITY.md](SECURITY.
 
 Requirement|Notes
 ---|---
-Go version | Go1.11.4 or higher
```

Comment:
Do we require 1.12 though? I thought running with 1.11.4 is still ok

Reply:
You are right. 1.11.4 is still OK. It's just we only test with 1.12 from this PR on.

Reply:
The fact that we only test 1.12 should not force us to require 1.12.

Reply:
OK, you are right. I can revert that change. Shouldn't we then still ensure that all tests etc. run on all versions we support (looks like currently we could run on 1.10 - 1.12 without any changes but we are only sure about the latest version we test on).

Reply:
We do need to test `release-X` branches that they compile with all versions we support. But that's not a priority and can be done separately.